src/docs/product/reference/search/searchable-properties/user-feedback.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: User Feedback Properties
+sidebar_order: 40
+description: "Learn more about searchable user feedback properties."
+---
+
+<Include name="searchable-properties/intro-for-user-feedback.mdx" />
+
+## Searchable Properties
+
+Below is a list of keys and tokens that can be used in the user feedback search.
+
+<DefinitionList>
+
+### `assigned`
+
+Returns issues assigned to the defined user(s) or team(s). Values can be a user ID (your email address), `me` for yourself, `none` for no assignee, `my_teams` or `#team-name` for teams you belong to.
+
+- **Type:** team or org user
+
+### `browser.name`
+
+Name of the browser.
+
+- **Type:** string
+
+### `device.brand`
+
+Brand of the device.
+
+- **Type:** string
+
+### `device.family`
+
+Family of the device. Typically, the common part of a model name across generations. For example, iPhone, Samsung Galaxy.
+
+- **Type:** string
+
+### `device.model_id`
+
+Internal hardware revision to identify the device exactly.
+
+- **Type:** n/a
+
+### `device.name`
+
+Name of the device.
+
+- **Type:** string
+
+### `dist`
+
+Distinguishes build or deployment variants of the same release of an application. For example, the dist can be the build number of an XCode build or the version code of an Android build.
+
+- **Type:** string
+
+### `environment`
+
+The environment that the event was first seen in.
+
+- **Type:** string
+
+### `id`
+
+The feedback ID.
+
+- **Type:** UUID
+
+### `is`
+
+The properties of an issue. Values can be: `unresolved`, `resolved`, `assigned`, `unassigned`, `linked`, or `unlinked`. The `linked` and `unlinked` values return issues based on whether they are linked to an external issue tracker or not.
+
+- **Type:** status
+
+### `level`
+
+The severity of the user feedback. If the feedback came in through the crash-report modal, then the level would be `error`; otherwise, the level is `info`.
+
+- **Type:** string
+
+### `os.name`
+
+The name of the operating system.
+
+- **Type:** string
+
+### `sdk.name`
+
+Name of the Sentry SDK that sent the event.
+
+- **Type:** string
+
+### `sdk.version`
+
+Version of the Sentry SDK that sent the event.
+
+- **Type:** string
+
+### `timestamp`
+
+The finish timestamp of the transaction. Returns events with matching datetime.
+
+- **Type:** datetime
+
+### `transaction`
+
+The error or transaction name identifier.
+
+- **Type:** string
+
+### `user.email`
```

Comment:
We can't search by user email if the email was entered by the user inside the feedback widget. I think it's important to note that this email search is specifically if we get the user email from the `Sentry.setUser` SDK option as a tag. cc/ @bruno-garcia

Reply:
adjusted wording but not sure the best way to phrase this

Reply:
me too, that's why i tagged bruno in hopes he would have a good way to convey this 😭